.sample.env

```diff
@@ -32,7 +32,7 @@ PORT=5000
 
 # Flask Environment
 
-FLASK_APP=fbosbot
+FLASK_APP=run.py
```

Comment:
`fbosbot` is the package name and directly reference the fbosbot/__init__.py where the create_app() live. 
Look at the default flask tutorial to learn more about that : https://flask.palletsprojects.com/en/1.1.x/tutorial/factory/

Reply:
@elinguiuriel Thanks for reviewing the PR!

Yes, you are correct but, we don't have any entry point to instantiate our project so, we need an entry point to run the project (i.e. call the [create_app()](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/src/fbosbot/__init__.py#L14) view).

This can be done in various ways:-
1. Use run.py file (as I done in this PR)
2. Use wsgi.py file itself to call create_app()
3. Don't use any external files and just call the create_app() view from where it lies itself.

Method-1 and 2 are recommended ways. I used Method-1 in this PR. Let me know what you think.

Reply:
@PrashanthPuneriya we fix #21 by adding `python-dotenv` in [requirements.txt](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/requirements.txt)

The entry point is `fbosbot/__init__.py` referenced by the package name `fbosbot`

Look at the [Dockerfile](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/Dockerfile) to see how we launch the app in a production server using Gunicorn:

```
gunicorn "fbosbot:create_app()" --worker-class gevent --workers 5 --bind 0.0.0.0:$PORT --max-requests 10000 --timeout 5 --keep-alive 5 --log-level info
```
You can also deploy by using `waitress` like described [here](https://flask.palletsprojects.com/en/1.1.x/tutorial/deploy/)

I want to stick to the way of doing described in the Flask official tutorial. So it will be easy for a new developer to only learn the tutorial to be able to contribute

Reply:
@elinguiuriel If that is the case then, we no longer need [wsgi.py](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/src/wsgi.py) file. It is just doing what I said in the above comments but, its implementation is wrongly done.

Reply:
@PrashanthPuneriya I am really interested to learn more about the recommended way of doing stuff.
Can you share a link were a can see that the methods 1) and 2) you presented are recommended ?

Reply:
@elinguiuriel Oh! Extremely sorry, for quoting them as `recommended ways`. I just picked it up from somewhere else and went with the flow and didn't introspected them as part of learning. Thanks for informing me and pointing out my mistake that it is not recommended way.

Reply:
@elinguiuriel I was wondering to create a PR to either fix or delete [wsgi.py](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/src/wsgi.py) file. What do you think?

Reply:
@PrashanthPuneriya there is no wsgi.py in this project! 
We will create a documentation folder at that time it will be great if you could help explain how to deploy the project using WSGI
.
What do you think about picking another issue [HERE](https://github.com/fbdevelopercircles/open-source-edu-bot/issues) ? 😌

Reply:
I think you are mistaken. There is a wsgi.py file. Click on this [link to view wsgi.py](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/src/wsgi.py) file of this project.

Reply:
Oh, before refactoring we used a single file project with wsgi and app.py as entry point, I forget to delete that wsgi.py file that's not used anywere. Sorry for mysleading by letting that file in the repo.

Reply:
@elinguiuriel I would love to fix that then!

Reply:
Go ahead ! Also don't forget you can pick many other issues